app/templates/index.html

```diff
@@ -6,10 +6,16 @@
 			{% with m = get_flashed_messages(with_categories=true) %}
 				{% if m %}
 					{% for cat,msg in m %}
-						<div class="flash-{{cat}}">{{msg}}</div>
+						<div class="flash-{{cat}}">Your badges has been successfully generated.</div>
```

Comment:
This will show "Your badges has been successfully generated." for the error msg as well